cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -1043,3 +1043,25 @@ func ExtractControlPlaneMachines(machines []*clusterv1.Machine) ([]*clusterv1.Ma
 	}
 	return controlPlaneMachines, nodes, nil
 }
+
+// ExtractControlPlaneMachineDeployments separates machinedeployments defining control plane machines from those defining nodes.
+// This is currently done by looking at which machinedeployments specify machine templates with a control plane version
+func ExtractControlPlaneMachineDeployments(machineDeployments []*clusterv1.MachineDeployment) (*clusterv1.MachineDeployment, []*clusterv1.MachineDeployment, error) {
+	nodeDeployments := []*clusterv1.MachineDeployment{}
+	controlPlaneDeployments := []*clusterv1.MachineDeployment{}
+	for _, machineDeployment := range machineDeployments {
+		if util.IsControlPlaneMachineDeployment(machineDeployment)) {
```

Comment:
Replace `))` with `)` in `IsControlPlaneMachineDeployment(machineDeployment))`

Reply:
got it :)